tests/bench_pertensor_fp8_old_vs_new.py

```
@@ -0,0 +1,202 @@
+New path: BF16 → fp8_blockwise_gemm_sm90(BF16, FP8, weight_scale)  (no activation quant)
+
+Controlled via AS_FP8_NO_CUTLASS=1 env var which forces the old path.
+
+Usage:
+  pip install dist/dashinfer-*.whl --force-reinstall --no-deps
+  CUDA_VISIBLE_DEVICES=0 python tests/bench_pertensor_fp8_old_vs_new.py \
+      --model_path /scratch/jzhang/models/gpt-oss-20b
+"""
+
+import argparse
+import gc
```

Flake8 (7.3.0)

[error] 'gc' imported but unused

(F401)